ui/components/app/whats-new-modal/whats-new-modal.tsx

```diff
@@ -106,25 +111,40 @@ export default function WhatsNewModal({ onClose }: WhatsNewModalProps) {
     onClose();
   };
 
+  const handleCreateSolanaAccount = () => {
+    setShowCreateSolanaAccountModal(true);
+  };
+
   return (
-    <Modal
-      onClose={handleModalClose}
-      data-testid="whats-new-modal"
-      isOpen={notifications.length > 0}
-      isClosedOnOutsideClick
-      isClosedOnEscapeKey
-    >
-      <ModalOverlay />
-
-      {notifications.map(({ id }) => {
-        const notification = getTranslatedUINotifications(t)[id];
-
-        return renderNotification({
-          notification,
-          onClose,
-          onNotificationViewed: handleNotificationViewed,
-        });
-      })}
-    </Modal>
+    <>
+      <Modal
+        onClose={() => null}
+        data-testid="whats-new-modal"
+        isOpen={notifications.length > 0 && !showCreateSolanaAccountModal}
+        isClosedOnOutsideClick
+        isClosedOnEscapeKey
+      >
+        <ModalOverlay />
+
+        {notifications.map(({ id }) => {
+          const notification = getTranslatedUINotifications(t)[id];
+
+          return renderNotification({
+            notification,
+            onClose,
+            onNotificationViewed: handleNotificationViewed,
+            onCreateSolanaAccount: handleCreateSolanaAccount,
+          });
+        })}
+      </Modal>
+      {showCreateSolanaAccountModal && (
+        <CreateSolanaAccountModal
+          onClose={() => {
+            setShowCreateSolanaAccountModal(false);
+            handleModalClose();
+          }}
+        />
+      )}
```

Comment:
Why is this in parallel with the normal notifications Modal?

Reply:
Well it technically is a part of the flow from the notification modal, but I didn't want to use the notification modal itself, as it's a bit limited in terms of the structure (header, content and footer) and it was easier to add it next to it and just hide notification when user decides to add new account.